llvm/test/CodeGen/WebAssembly/offset.ll

```diff
@@ -666,3 +666,29 @@ define {i32,i32,i32,i32} @aggregate_return() {
 define {i64,i32,i16,i8} @aggregate_return_without_merge() {
   ret {i64,i32,i16,i8} zeroinitializer
 }
+
+;===----------------------------------------------------------------------------
```

Comment:
I didn't add many tests here, since it seems pretty well covered for other load patters.